components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -1124,7 +1126,9 @@ public static void initTokenExpiryTimesOfSps(int tenantId) {
      * @param consumerKey
      * @param tenantId
      * @return A SpOAuth2ExpiryTimeConfiguration Object
+     * @deprecated due to UI implementation
```

Comment:
If there's a new method to get ExpiryTimeConfiguration it should be mentioned here.

Reply:
Fix with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/477